addons/project/models/project.py

```diff
@@ -2066,11 +2066,12 @@ def action_dependent_tasks(self):
         action = {
             'res_model': 'project.task',
             'type': 'ir.actions.act_window',
-            'context': {**self._context, 'default_depend_on_ids': [Command.link(self.id)]},
+            'context': {**self._context, 'default_depend_on_ids': [Command.link(self.id)], 'show_project_update': False},
         }
         if self.dependent_tasks_count == 1:
             action['view_mode'] = 'form'
             action['res_id'] = self.dependent_ids.id
+            action['views'] = [(self.env.ref('project.view_task_form2').id, 'form')]
```

Comment:
I think it is not necessary to ref the form view. I think it is the default one for this model. 🙂 
Can you check it? Thanks in advance!
```suggestion
            action['views'] = [False, 'form')]
```

Reply:
Done!